ui-ngx/src/app/shared/models/rule-node.models.ts

```diff
@@ -358,13 +366,13 @@ export enum MessageType {
   ENTITY_UNASSIGNED = 'ENTITY_UNASSIGNED',
   ATTRIBUTES_UPDATED = 'ATTRIBUTES_UPDATED',
   ATTRIBUTES_DELETED = 'ATTRIBUTES_DELETED',
+  ALARM_ACKNOWLEDGED = 'ALARM_ACKNOWLEDGED',
+  ALARM_CLEARED = 'ALARM_CLEARED',
+  ENTITY_ASSIGNED_FROM_TENANT = 'ENTITY_ASSIGNED_FROM_TENANT',
+  ENTITY_ASSIGNED_TO_TENANT = 'ENTITY_ASSIGNED_TO_TENANT',
   TIMESERIES_UPDATED = 'TIMESERIES_UPDATED',
   TIMESERIES_DELETED = 'TIMESERIES_DELETED',
-  RPC_QUEUED = 'RPC_QUEUED',
-  RPC_DELIVERED = 'RPC_DELIVERED',
-  RPC_SUCCESSFUL = 'RPC_SUCCESSFUL',
-  RPC_TIMEOUT = 'RPC_TIMEOUT',
-  RPC_FAILED = 'RPC_FAILED'
+  FAILURE = 'FAILURE'
```

Comment:
Hi @kalutkaz and @vvlladd28, if I'm not mistaken, there is no 'Failure' message type used in a rule engine. Failure is a relation type that might be used for the output messages.